### ISSUE TITLE
Check if there is history to goBack() to?

### ISSUE THREAD

**Issue by marko1943** [kelsonpw/react-chrome-extension-router#4]
Hello,

First of all, thanks for this awesome and easy to use router.

My question is, is there a way to check if there is a history to go back to?
So i can, for example, conditionally render the back arrow and use the `goBack()` function?

Also, I'm using React 16.12. with chrome extensions, and `getCurrent()` function always returns **undefined** for me.

eg. this
`
const Popup = () => {
  const current = getCurrent();
  console.log(current);
};
`

logs undefined.

Regards,
Marko

**Comment by kelsonpw** [comment]
Hey @marko1943, thanks for creating this issue!

I just updated the library to v1.1.0.  It should now include a new method called `getComponentStack` that will return an array of all the components on the stack.

I also updated `getCurrent` to return `{component: false, props: null}` when no components are on the stack.

I believe getCurrent was returning undefined for you because no components had been added to the stack.  If the direct child of `<Router>here</Router>` is what is currently being rendered, both getCurrent and getComponentStack will empty.  So basically getCurrent will only work if something is on the stack, otherwise it will return `{component: false, props: null}`.
